topology/Internet2/Internet2Manhattan/I2ManhattanInfrastructure.yaml

```
@@ -10,14 +10,14 @@ Resources:
     ContactLists:
       Administrative Contact:
         Primary:
-          Name: Diego Davila
-          ID: bc36e7fe84fffcb5cf195fe09cc42336f5cd5d1f
+          Name: Fabio Andrijauskas
+          ID: ef5d0be3ec1df913935853ed82396b1eb37a7db0
       # Security Contact is one to three people to contact regarding security issues
       Security Contact:
         Primary:
-          Name: Diego Davila
-          ID: bc36e7fe84fffcb5cf195fe09cc42336f5cd5d1f
-    FQDN: osg.newy32aoa.nrp.internet2.edu
+          Name: Fabio Andrijauskas
+          ID: ef5d0be3ec1df913935853ed82396b1eb37a7db0
+    FQDN: osg-new-york-stashcache.nrp.internet2
```

```suggestion
    FQDN: osg-new-york-stashcache.nrp.internet2.edu
```
I think the .edu got dropped by accident?

---

sorry